routes/api.js

```diff
@@ -0,0 +1,103 @@
+import express from 'express';
```

Comment:
I think finally this file shouldn't exist ✌️

Reply:
I moved logic from app.js to this file , and removed app.js instead api.js